tests/test_data/test_user_security_utils.py

```diff
@@ -18,6 +18,7 @@
 # CRUD SETUP
 class User(Base):
     __tablename__ = 'user_security'
+    __table_args__ = {'extend_existing': True}

```

Comment:
What is the reason for this?

Reply:
no the table structure is not changing, the main error was this `Table 'user_security' is already defined for this MetaData instance` and this is also imported once in the code and db is created once so this was the only solution I found for this issue that could fix it